extensions_admin/setup.py

```diff
@@ -3,7 +3,7 @@
 
 setup(
     name='pulp_python_extensions_admin',
-    version='0.0.0',
+    version='1.0.0',
```

Comment:
I don't personally think it's necessary for the setup.py files to have this, but if we have a team policy that it should I can change it later. For now, I'll leave it as is since primarily the spec file will communicate its "alphaness".